react/features/invite/components/AddPeopleDialog.web.js

```diff
@@ -229,11 +235,17 @@ class AddPeopleDialog extends Component {
                 addToCallInProgress: true
             });
 
+            this.props._conference
+                && inviteRooms(
+                    this.props._conference,
+                    this.state.inviteItems.filter(
+                        i => i.type === 'videosipgw'));
+
             invitePeople(
                 this.props._inviteServiceUrl,
                 this.props._inviteUrl,
                 this.props._jwt,
-                this.state.inviteItems)
+                this.state.inviteItems.filter(i => i.type === 'user'))
             .then(() => {
```

Comment:
Would using a Promise.all make sense in this case so that addToCallInProgress is set to false only after both inviteRooms and invitePeople complete?

Reply:
inviteRooms currently is sync and not a Promise